metricbeat/Makefile

```diff
@@ -34,8 +34,8 @@ configs: python-env
 	@cat ${ES_BEATS}/metricbeat/_meta/common.reference.yml > _meta/beat.reference.yml
 	@${PYTHON_ENV}/bin/python ${ES_BEATS}/script/config_collector.py --beat ${BEAT_NAME} --full $(PWD) >> _meta/beat.reference.yml
 	@rm -rf modules.d
-	${PYTHON_ENV}/bin/python ${ES_BEATS}/script/modules_collector.py --beat ${BEAT_NAME} --docs_branch=$(DOCS_BRANCH)
-	@chmod go-w modules.d/*
+	mage config
+	@chmod go-w ${ES_BEATS}/metricbeat/modules.d/*
```

Comment:
@eloyekunle I have been testing this branch, and it doesn't work well for generators, the change in this chmod shouldn't be needed, this `modules.d` directory should be generated by `make configs` also in generated beats.

I think that we should change the magefile in `generator/metricbeat/{beat}/magefile.go` so it does something similar of what mage does for the real Metricbeat.

It would be to rename the `Config()` function to something like `configYML()` and then implement `Config()` as:
```
func Config() {
        mg.Deps(configYML, metricbeat.GenerateDirModulesD)
}
```

Could you revert this change to the chmod and give a try to this proposal?

Reply:
@jsoriano I have made the change to `generator/metricbeat/{beat}/magefile.go`.